### PR TITLE
Updated free-courses-bn.md

### PR DESCRIPTION
**Pull request by Asrarul-Hoque-Eusha**
Added Django and Restapi courses.

## What does this PR do?
Add resources.

### Description
Django and Restapi Bengali Course Tutorial.

### Why is this valuable (or not)?
One can learn Django and Restapi in Bengali and use them further.

### How do we know it's really free?
All of them are youtube playlists, that's why it is free.

### For book lists, is it a book? For course lists, is it a course? etc.
They are Courses.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -73,6 +75,11 @@ Shikkhangon BD
 * [Dart All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVC2-nXUrvpcQEz3FgCSIQHT) - Stack Learner
 
 
+### Django
```
Comment (shrn01) [inline courses/free-courses-bn.md:78]: Can you move Django under python?

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Review by Asrarul-Hoque-Eusha — COMMENTED** [review]
I have moved Django to Python.

File: courses/free-courses-bn.md (hunk 1 above)
Comment (Asrarul-Hoque-Eusha) [reply courses/free-courses-bn.md]: I have moved Django to Python.

**Review by shrn01 — APPROVED** [review]

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -170,6 +172,11 @@ Shikkhangon BD
 * [React JS Tutorial Bangla Series for Beginners](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9M6HVQv7a36cp8LKzyHIePr) - Sumit Saha (Learn with Sumit)
 
 
+### Restapi
```
Comment (shrn01) [inline courses/free-courses-bn.md:175]: Move this after python

File: courses/free-courses-bn.md [hunk 3]
```diff
@@ -221,6 +223,16 @@ Shikkhangon BD
...
 
 
+#### Django
+
+* [Django Bangla Tutorial](https://www.youtube.com/playlist?list=PLbC4KRSNcMnqUp_v1nxbQaoImTN3kWS_V) - Learn Hunter
+
+
+### Restapi
+
+* [Restapi Bangla Tutorial](https://www.youtube.com/playlist?list=PLbC4KRSNcMnoQONzuNtFlhEzegTYadoBY) - Learn Hunter
```
Comment (eshellman) [inline courses/free-courses-bn.md:233]: this seems to be a laravel tutorial

File: courses/free-courses-bn.md [hunk 4]
```diff
@@ -25,8 +25,11 @@
 * [MongoDB](#mongodb)
 * [MySQL](#mysql)
 * [PHP](#php)
+    * [Laravel](#laravel)
 * [Programming paradigms](#programming-paradigms)
 * [Python](#python)
+    * [Django](#django)
+* [Restapi](#restapi)
```
Comment (shrn01) [inline courses/free-courses-bn.md:32]:
```suggestion
```

Remove this line

**Pull request merged**